charts/quickwit/values.yaml

```diff
@@ -277,20 +313,26 @@ janitor:
 
   # Extra env for janitor
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed